usl_pipeline/usl_lib/tests/chunkers/test_elevation_chunkers.py

```diff
@@ -0,0 +1,135 @@
+import tempfile
+
+import numpy
+from numpy import testing
+import rasterio
+
+from usl_lib.readers import elevation_readers
+from usl_lib.chunkers import elevation_chunkers
+from usl_lib.shared import geo_data
+from usl_lib.shared import suppliers
+
+
+def prepare_test_geotiff_elevation_memory_file(file_path: str) -> None:
```

Comment:
It looks like this is no longer a memory file?

Reply:
Sorry, forgot to update it for some reason...